services/order.py

```diff
@@ -0,0 +1,46 @@
+from django.db import transaction
+from db.models import Order, Ticket, User, MovieSession
+from django.utils import timezone
+from django.core.exceptions import ObjectDoesNotExist
+from django.db.models import QuerySet
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],

```

Comment:
The type hint 'list[dict]' is used here, which is correct for Python 3.9 and above. Ensure that your environment supports this syntax. If not, consider using 'List[Dict]' from the 'typing' module.